File: src/main/kotlin/MyStrategy.java
import model.Game;
import model.Move;
import model.Wizard;
import model.World;

public final class MyStrategy implements Strategy {

    private StrategyManager strategyManager;

    @Override
    public void move(Wizard self, World world, Game game, Move move) {
        initialize();
        strategyManager.nextTick(self, world, game, move);
    }

    private void initialize() {
        FindHelper.clearCache();

        if (strategyManager == null) {
            strategyManager = new StrategyManager();
        }
    }

}

File: src/main/kotlin/MyStrategy.kt
import model.Game
import model.Move
import model.Wizard
import model.World

//v15
class MyStrategy : Strategy {

    private var strategyManager: StrategyManager? = null

    override fun move(self: Wizard, world: World, game: Game, move: Move) {
        initialize()
        strategyManager!!.nextTick(self, world, game, move)
    }

    private fun initialize() {
        FindHelper.clearCache()

        if (strategyManager == null) {
            strategyManager = StrategyManager()
        }
    }

}